client/src/views/active/startup-view.tsx

```diff
@@ -63,13 +71,33 @@ export default function StartUpView(props: StartUpViewProps) {
 
 
 	useEffect(() => {
+		console.log('active rocket:', activeContext.rocket);
+		console.log('active mission:', activeContext.activeMission);
 		getDataConfig();
-	}, [getDataConfig]);
+	}, []);
+
+	const savePacketToDatabase = useCallback(async (packet: IAprsTelemetryPacket) => {
+		await saveAprsPacket(activeContext.activeMission, packet, packetNumber);
+	}, [currentPacket]);
 
 	useEffect(() => {
 		if (sc.aprsPacket.Data === undefined) return 
-		setLog(prev => [...prev, JSON.stringify(sc.aprsPacket.Data) as string || '']);
+		activeContext.updateLogs(sc.aprsPacket.Data as IAprsTelemetryPacket);
+		setCurrentPacket(sc.aprsPacket.Data as IAprsTelemetryPacket);
+	}, [sc.aprsPacket.Data, activeContext]);
+
+	useEffect(() => {
+		setPacketNumber(packetNumber + 1);
+		try {
+			savePacketToDatabase(currentPacket)
```

Comment:
Related to another comment: why try save packet if we have no mission id, which would cause this to fail?